Sources/Fullscreen/AdConfirmationView/Views/AdConfirmationObjectView.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  Copyright © 2019 FINN AS. All rights reserved.
+//
+
+public class AdConfirmationObjectView: UIView {
+    private lazy var imageView: RemoteImageView = {
+        let imageView = RemoteImageView(withAutoLayout: true)
+        imageView.dataSource = self
+        imageView.clipsToBounds = true
+        imageView.layer.cornerRadius = 8.0
+        return imageView
+    }()
+
+    private lazy var titleLabel: Label = {
+        let titleLabel = Label(style: .title3Strong, withAutoLayout: true)
+        titleLabel.numberOfLines = 0
+        titleLabel.textAlignment = .center
+    return titleLabel
```

Comment:
Indentation is a little off 🙃

Reply:
Hahahaha, this is the greatest indentation mistake ever